plugins/aws/sns/snsTopicExposed.js

```diff
@@ -0,0 +1,91 @@
+
+var async = require('async');
+var helpers = require('../../../helpers/aws');
+
+module.exports = {
+    title: 'SNS Topic Exposed',
+    category: 'SNS',
+    domain: 'Application Integration',
+    severity: 'High',
+    description: 'Ensures SNS topics are not publicly accessible.',
+    more_info: 'Allowing anonymous users to have access to your Amazon SNS topics can lead to unauthorized actions such as intercepting and receiving/publishing messages without permission.',
```

Comment:
```suggestion
    more_info: 'Allowing anonymous users to have access to your Amazon SNS topics can lead to unauthorized actions such as intercepting and receiving/publishing messages without permission. To avoid data leakage and unexpected costs , limit access to SNS topics by implementing the right permissions.',
```